src/lib/logger.ts

```diff
@@ -46,22 +54,22 @@ const loggerMiddleware = function loggerMiddleware(
 
 const supplementProductionInfo = format((info) => {
   // LOGSTASH 675 versioning https://gds-way.cloudapps.digital/manuals/logging.html
-  const LOG_VERSION = 1
   const productionContext: any = {
     toolbox_id: session.get(TOOLBOX_ID_KEY),
-    correlation_id: session.get(CORRELATION_ID_KEY),
+    x_request_id: session.get(CORRELATION_ID_KEY),
     user_id: session.get(AUTHENTICATED_USER_ID_KEY)
   }
-  productionContext['@version'] = LOG_VERSION
   return Object.assign(info, productionContext)
 })
 
 if (!config.common.development) {
   const productionTransport = new transports.Console({
     format: combine(
       supplementProductionInfo(),
-      timestamp({ format: 'YYYY-MM-DDTHH:mm:ss.sssZ', alias: '@timestamp' }),
-      format.json()
```

Comment:
`process.env.ENVIRONMENT` is available through the Sentry configuration -- I wonder if it's worth bringing it into something like 'server' or 'common' and having both of these use that

Reply:
Good idea, I've done that